js/abp-preprocessed.es6.js

```diff
@@ -4,64 +4,72 @@
  *
  * This will be browserifyed and turned into abp.js by running 'grunt'
  */
-abp = require('abp-filter-parser');
+abp = require('abp-filter-parser')
 const ONEDAY = 1000*60*60*24
 
-easylists = {
-    privacy: {
-        url: 'https://duckduckgo.com/contentblocking.js?l=easyprivacy',
-        parsed: {},
+let lists = {
+    easylists : {
+        privacy: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=easyprivacy',
+            parsed: {},
+        },
+        general: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=easylist',
+            parsed: {},
+        }
     },
-    general: {
-        url: 'https://duckduckgo.com/contentblocking.js?l=easylist',
-        parsed: {},
+    whitelists: {
+        preWhitelist: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=trackers-whitelist',
+            parsed: {}
+        }
     }
-};
+}
+
+easylists = lists.easylists
+whitelists = lists.whitelists
 
 /*
  * Get the list data and use abp to parse.
  * The parsed list data will be added to 
  * the easyLists object.
  */
 function updateLists () {
-    for (let list in easylists) {
-        let url = easylists[list].url
-        let atb = settings.getSetting('atb')
-        let set_atb = settings.getSetting('set_atb')
-        let versionParam = getVersionParam()
-        let etag = settings.getSetting(list + '-etag')
-
-        if (atb) url += '&atb=' + atb
-        if (set_atb) url += '&set_atb=' + set_atb
-        if (versionParam) url += versionParam
-
-        console.log("Checking for list update: ", list)
-
-        // if we don't have parsed list data skip the etag to make sure we
-        // get a fresh copy of the list to process
-        if (Object.keys(easylists[list].parsed).length === 0) etag = ''
-
-        load.loadExtensionFile({url: url, source: 'external', etag: etag}, (listData, response) => {
-            let newEtag = response.getResponseHeader('etag')
-
-            console.log("Updating list: ", list)
-        
-            // sync new etag to storage
-            settings.updateSetting(list + '-etag', newEtag)
-
-            abp.parse(listData, easylists[list].parsed)
-            easylists[list].loaded = true;
-        });
-
+    let atb = settings.getSetting('atb')
+    let set_atb = settings.getSetting('set_atb')
+    let versionParam = getVersionParam()
```

Comment:
Let's be judicious with `let` vs `const` throughout this PR. If something is going to be redefined later, use `let`. If not, use `const`.